DESCRIPTION

```diff
@@ -49,7 +50,8 @@ Suggests:
     rmarkdown,
     roxygen2,
     spelling,
-    testthat (>= 3.0.0)
+    testthat (>= 3.0.0),
+    withr
```

Comment:
{withr} is an intentional new Suggest?

Reply:
Yup, I had to add `withr` calls into `testthat`